iina/MenuController.swift

```diff
@@ -137,14 +137,14 @@ class MenuController: NSObject, NSMenuDelegate {
 
     // -- aspect
     var aspectList = AppData.aspects
-    aspectList.insert("Default", at: 0)
+    aspectList.insert(NSLocalizedString("quicksetting.aspect_item." + PlayerCore.shared.info.unsureAspect, comment: "Default"), at: 0)
     bind(menu: aspectMenu, withOptions: aspectList, objects: nil, objectMap: nil, action: #selector(MainWindowController.menuChangeAspect(_:))) {
```

Comment:
It maybe too risky to do so. I agree with your opinion:

> But I think the better way to select a menu item is using id,not the title of that item because the title may be translated.
> So,why don't we use English for coding and localized string for displaying?

So please discard all "Default" related changes temporarily and let me fix it later.

Sorry for that!